pom.xml

```diff
@@ -78,6 +78,10 @@
             <version>${problem-spring-web.version}</version>
         </dependency>
         <dependency>
+            <groupId>org.springframework.boot</groupId>
```

Comment:
why create this new dependency? I didn't find any import for the hateoas spring API